build/grunt.js

```diff
@@ -239,38 +303,25 @@ module.exports = function(grunt) {
       }
     },
     browserify: {
-      options: {
-        browserifyOptions: {
-          debug: true,
-          standalone: 'videojs'
-        },
-        plugin: [
-          ['browserify-derequire']
-        ],
-        transform: [
-          require('babelify').configure({
-            sourceMapRelative: './',
-            loose: ['all']
-          }),
-          ['browserify-versionify', {
-            placeholder: '__VERSION__',
-            version: pkg.version
-          }],
-          ['browserify-versionify', {
-            placeholder: '__VERSION_NO_PATCH__',
-            version: version.majorMinor
-          }],
-          ['browserify-versionify', {
-            placeholder: '__SWF_VERSION__',
-            version: pkg.dependencies['videojs-swf']
-          }]
-        ]
-      },
+      options: browserifyGruntOptions(),
       build: {
         files: {
           'build/temp/video.js': ['src/js/video.js']
         }
       },
+      dist: {
+        options: browserifyGruntOptions({
+          transform: [
+            ['browserify-versionify', {
+              placeholder: '../node_modules/vtt.js/dist/vtt.js',
+              version: 'https://raw.githubusercontent.com/gkatsev/vtt.js/vjs-v0.12.1/dist/vtt.min.js'
```

Comment:
Actually, we can't link from githubusercontent. This URL should work, though: https://cdn.rawgit.com/gkatsev/vtt.js/vjs-v0.12.1/dist/vtt.min.js